app/src/main/java/org/oppia/app/profile/AddProfileActivityPresenter.kt

```diff
@@ -79,6 +80,17 @@ class AddProfileActivityPresenter @Inject constructor(
 
     addTextChangedListeners(binding)
     addButtonListeners(binding)
+
+    binding.inputName.setInput(profileViewModel.savedName)
```

Comment:
Instead of setting data to xml using binding, use viewModel. Check how `createPin` is getting used in xml from `ViewModel`. In this presenter you can just set data in viewmodel and then viewmodel will set data to the xml file.

Reply:
ok